lib/components/select/styles.js

```diff
@@ -19,7 +19,7 @@ const multiValueRemoveStyles =
 const clearIndicatorStyles = 'es-uic-text-gray-500 es-uic-p-1 es-uic-rounded-md hover:bg-red-50 hover:text-red-800 es-uic-transition';
 const dropdownIndicatorStyles =
 	'es-uic-text-gray-500 es-uic-px-1 group-hover:es-uic-text-black [&>svg]:es-uic-transition-transform [&>svg]:es-uic-duration-500 [&>svg]:es-uic-size-5.5';
-const menuStyles = 'es-uic-z-50 es-uic-rounded-md es-uic-border es-uic-border-gray-200 es-uic-bg-white es-uic-shadow-lg es-uic-mt-1 es-uic-text-sm es-uic-overflow-x-hidden';
+const menuStyles = 'es-uic-rounded-md es-uic-border es-uic-border-gray-200 es-uic-bg-white es-uic-shadow-lg es-uic-mt-1 es-uic-text-sm es-uic-overflow-x-hidden';
```

Comment:
👀 Going back to the faulty version?

Reply:
Z-index never did anything, so no point in having it 😄 
The actual solution is in the commit 😉